### PR TITLE
Added Open Service Mesh Lab

### LINKED ISSUE #2189: [Learn] Add Open Service Mesh lab
Current Behavior
The newly created Open Service Mesh lab hasn't been added to https://layer5.io/learn .

Desired Situation
Let's put up the Open Service Mesh lab [here](https://layer5.io/learn/service-mesh-labs) .

https://github.com/layer5io/layer5/blob/master/src/sections/Learn/Service-Mesh-Labs/courseData.js

https://github.com/layer5io/layer5/tree/master/src/collections/service-mesh-labs

Contributor Resources

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the master branch.

See contributing instructions
See Layer5 site designs in this Figma project. Join the Layer5 Community for access.

### PR DESCRIPTION
**Pull request by khanjasir90**
Signed-off-by: khanjasir90 <khanmohdjasir@gmail.com>

**Description**

This PR fixes #2189 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/service-mesh-labs/working_with_meshery_and_traefik/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Working with Meshery and Traefik"
```
Comment (leecalcote) [inline src/collections/service-mesh-labs/working_with_meshery_and_traefik/index.mdx:2]: OSM or Traefik?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: src/collections/service-mesh-labs/working_with_meshery_and_traefik/index.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+<div id="katacoda-scenario"
+  data-katacoda-id="layer5/courses/meshery-adapters/traefik-meshery-adapter"
```
Comment (warunicorn19) [inline src/collections/service-mesh-labs/working_with_meshery_and_traefik/index.mdx:6]:
```suggestion
  data-katacoda-id="layer5/courses/meshery-adapters/open-service-mesh-meshery-adapter"
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
@leecalcote this is my fault, I wrote Traefik mesh in one of the lines in the issue.

@khanjasir90 it seems like you're not on the latest, the traefik mesh lab has already been added. if could make the changes accordingly for OSM

File: src/collections/service-mesh-labs/working_with_meshery_and_traefik/index.mdx (hunk 1 above)
Comment (khanjasir90) [reply src/collections/service-mesh-labs/working_with_meshery_and_traefik/index.mdx:2]: @leecalcote  so sorry I created a wrong PR!

**Pull request closed without merging**